.github/workflows/python-versions-runner.yml

```diff
@@ -0,0 +1,58 @@
+name: Python versions runner
+on:
+  workflow_dispatch:
+    inputs:
+      versions:
+        description: 'Versions to build'
+        required: true
+        default: '["", ""]'
+      publish-releases:
+        description: 'Whether to publish releases'
+        required: true
+        default: 'false'
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  trigger_builds:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ${{ fromJson(github.event.inputs.versions) }}
+    steps:
+      - run: |
+          $payload = @{
+            "ref" = "main"
+            "inputs" = @{
+              "VERSION" = "${{ matrix.version }}"
+              "PUBLISH_RELEASES" = "false"
+            }
+          } | ConvertTo-Json
+          $headers = @{
+            Authorization="Bearer ${{ secrets.PERSONAL_TOKEN }}"
+          }
+          Invoke-RestMethod -uri "$env:GITHUB_API_URL/repos/$env:GITHUB_REPOSITORY/actions/workflows/python-builder.yml/dispatches" -method POST -headers $headers -body $payload
+          Start-Sleep -seconds 40
```

Comment:
Can we get rid of this and set up a cycle with retries? Setting such magic values seems unreliable

Reply:
Updated